ui/app/AppLayouts/Communities/panels/MintTokensSettingsPanel.qml

```diff
@@ -140,71 +135,65 @@ StackView {
 
     }
 
-    initialItem: mintedTokensViewComponent
-
-    Component {
-        id: tokenObjectComponent
-
-        TokenObject {}
-    }
+    initialItem: SettingsPage {
+        implicitWidth: 0
```

Comment:
why `0` here?

Reply:
I've just moved this code from a component to a direct assignment. Going to check that..

Reply:
On Qt 5.15.2 (which is still used for deployment on linux from what I know) there is binding loop on implicitWidth which can be fixed by setting it to `0` explicitly. It's safe bc implicitWidth is not used here anyway.